.github/workflows/integration-tests.yml

```diff
@@ -26,7 +26,9 @@ jobs:
       matrix:
         browser: ["chrome"]
     name: Cypress tests on ${{ matrix.browser }}
-
+    env:
+      MONGO_DB_NAME: 'covid-19'
```

Comment:
Doesn't matter much as we are not using this for COVID-19, but the DB name we were using was `covid19`